odoo/addons/base/static/src/js/res_config_settings.js

```diff
@@ -199,6 +200,17 @@ var BaseSettingRenderer = FormRenderer.extend({
      * @param {int} index
      */
     _moveToTab: function (index) {
+        //Used to reorder the settings after a search and a tab switch
+        //Thought it would be better for the UX if the order stayed the same as the one on the left panel
+        if(!this.initial_order && this.$el.find('.settings')[0]){
```

Comment:
you use 3 times the this.$el.find('.settings')[0] expression.  Also a lookup in the DOM is kind of expensive. You can simply do this:
```
var settingsNode = this.$('.settings')[0];
if (!this.initialTabElements && settingsNode[0]) {...
```